lib/matplotlib/tests/test_ft2font.py

```diff
@@ -55,15 +55,14 @@ def test_fallback_smoke():
         fig.savefig(io.BytesIO(), format=fmt)
 
 
+@skipif_font_missing(['WenQuanYi Zen Hei', 'Noto Sans CJK JP'])
```

Comment:
I don't like that this will skip both parameterized versions of this test if only one of the fonts is missing... The implementation where it is checked in the method is more verbose, but allows for these parameterizations to still test when only some fonts are available.